src/components/ContentOpenWith/__tests__/IntegrationPortal-test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import IntegrationPortal from '../IntegrationPortal';
+
+const INTEGRATION_CONTAINER_ID = 'integration-container';
+
+describe('components/ContentOpenWith/IntegrationPortal', () => {
+    const getWrapper = props => shallow(<IntegrationPortal {...props} />);
+
+    it('should use an existing container if possible', () => {
+        const container = document.createElement('div');
+        container.id = INTEGRATION_CONTAINER_ID;
+        document.body.appendChild(container);
+
+        const wrapper = getWrapper({
+            integrationWindow: {
+                // use normal document in our integrationWindow
+                document,
+            },
+        });
+
+        const instance = wrapper.instance();
+        expect(instance.containerElement).toEqual(container);
+    });
+
+    it('should create the container if needed, and assign an id ', () => {
+        const wrapper = getWrapper({
+            integrationWindow: {
+                // use normal document in our integrationWindow
+                document,
+            },
+        });
+
+        wrapper.instance();
+        const containerElement = document.querySelector('div');
+        expect(!!containerElement).toBeTruthy();
+        expect(containerElement.id).toEqual(INTEGRATION_CONTAINER_ID);
+    });
+
+    describe('componentDidMount()', () => {
+        it('should copy styles and append the container to the integration window', () => {
+            const wrapper = getWrapper({
+                integrationWindow: {
+                    // use normal document in our integrationWindow
+                    document,
+                },
+            });
+            const instance = wrapper.instance();
+            instance.containerElement = document.createElement('div');
+            instance.copyStyles = jest.fn();
+
+            instance.componentDidMount();
+
+            expect(instance.copyStyles).toBeCalled();
+            expect(!!document.querySelector('div')).toBeTruthy();
```

Comment:
The "truthy" here would lead me to believe that this explicit casting isn't necessary.